desktop/yast2-packager.desktop

```diff
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=Install/Remove Software
 GenericName=Install/Remove Software
-Exec=xdg-su -c "/sbin/yast2 sw_single %F"
```

Comment:
Strange enough, I checked that calling yast2 directly (without the wrapper script) works from Thunar (XFCE file manager) and I assume it works also from Gnome's file manager. But it fails from Dolphin (KDE-Plasma-Today's_name file manager).